ui_testing/pages/order_page.py

```diff
@@ -125,13 +125,7 @@ def create_existing_order(self, no='', material_type='', article='', contact='',
         
         for test_unit in test_units:
             self.set_test_unit(test_unit)    
-        if multiple_suborders > 0:
```

Comment:
easier to create 5 suborders, since all the suborders testcases needs to be tested starting from the 3rd suborder+ so this way, it duplicates the first suborder according to the number of orders i need, to start my test case